stock_secondary_unit/models/procurement_rule.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class ProcurementRule(models.Model):
+    _inherit = 'procurement.rule'
+
+    def _get_stock_move_values(self, product_id, product_qty, product_uom,
+                               location_id, name, origin, values, group_id):
+        res = super(ProcurementRule, self)._get_stock_move_values(
+            product_id, product_qty, product_uom, location_id, name, origin,
+            values, group_id
+        )
+        if values.get('sale_line_id', False):
```

Comment:
@sergio-teruel I think this part of code is out of scope of this module.

Could you fix this quickly or reverting the merge of this module as it breaks 11.0 branch ?

Reply:
This is why this depends on `sale_stock`, not only on `stock`.

Reply:
Nope. The only aim of this part of code is to fill in fields depending on sale_secondary_unit... That's why the code fails!

Reply:
This part doesn't activate if the value is not present on the dictionary, so no possible fail here.

Reply:
This kind of code that depends partially on other modules that are 'possibly' installed is not safe as it is impossible to write test for them. This should be avoided at all!

Reply:
Moreover, if you just install sale and this module, this part of code will be triggered. See #544

Reply:
I see now the problem. It's not on this line, but the next one. Fixing with a glue module.